app/src/main/java/org/sopt/do_sopt_compose/ui/components/profile/FriendsProfileMusic.kt

```diff
@@ -0,0 +1,68 @@
+package org.sopt.do_sopt_compose.ui.components.profile
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.res.imageResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import org.sopt.do_sopt_compose.ui.components.MelonMusic
+
+
+@Composable
+fun FriendsProfileMusic(
+    profileImageUrl: Int,
+    nickname: String,
+    music: String,
+    statusMessage: String?
```

Comment:
이 변수를 사용할 때, null이면 ""로 바꿔주는 코드인 것 같습니다.
그렇다면 기본값을 ""로 두고, 필요할 때만 변수에서 넣어주는 건 어떨까요?
```suggestion
    statusMessage: String = ""
```

Reply:
오호 그래도 되겠네요..! 아마 제가 이렇게 했던 이유는 FriendsProfileMusic를 임포트하면 기본값이 없으면 statusMessage를 입력할 수 있게 뜨게 해줘서 그러긴 했엇습니당